### DISCUSSION TITLE
Take multiple photos

### DISCUSSION THREAD

**Discussion by jcroucher** [ionic-team/capacitor#5080] [Q&A] [answered]
I was wondering if there is a way to allow the user to keep taking photos until they are finished.
Rather than closing the camera after the photo is taken, it would just keep going and adding to a list.

I see there is the Camera Preview plugin which sort of looks like what I am after, but it seems to use a video stream.
I need one that can take photos with full image data and in heic format on iOS.

**Comment by jcroucher** [comment, marked as answer]
Update: I have modified the Camera Preview plugin to do what I want.

**Reply by confuzeus** [reply]
@jcroucher Hi, did you send a PR for this feature?

**Reply by jcroucher** [reply]
Nah I didn't. The changes I made were quite specific to what I was doing and would probably break things for others.
I can't remember what changes I made but I can share a zip of the plugin if that would be of any use?

**Reply by confuzeus** [reply]
Sure thing my friend. I have the same problem right now so I could definitely learn from your ideas. You can link to the zip here or email me at michael@confuzeus.com

Thanks a lot.

**Reply by JamesTheTerry** [reply]
@jcroucher is your modification posted anywhere?

**Reply by jcroucher** [reply]
@JamesTheTerry please see attached. 
[camera-preview-modified.zip](https://github.com/ionic-team/capacitor/files/14404378/camera-preview-modified.zip)

It was ages ago so I can't remember what was done, I just struggled to remember which project it was for.
I just copied out of the project and added a few JS lines in Modified Usage.txt file which will hopefully help.

Also note that this had a number of other modifications so it may not be suitable but hopefully it can help.

**Reply by JamesTheTerry** [reply]
@jcroucher thanks for the zip, I'll take a look and see what I can do. I appreciate your help!
